subt/tools/run.py

```diff
@@ -170,7 +170,7 @@ def _run_sim(client, circuit, logdir, world, robots):
             f"robotName{n}:={name}",
             f"robotConfig{n}:={kind}"
         ]
-    sim = _create_docker(client, "sim", "osrf/subt-virtual-testbed:cloudsim_sim_latest", command, mounts, environment)
+    sim = _create_docker(client, "sim", "cloudsim:latest", command, mounts, environment)
```

Comment:
I wonder how not to forget to rebuildlocal `cloudsim` when external is upgraded?

Reply:
The solution is the same as we are using for our base image. We can start using specific versions instead of "latest".